packages/xo-web/src/xo-app/home/index.js

```diff
@@ -705,15 +708,17 @@ export default class Home extends Component {
       createFilter(() => this.props.items, this._getFilterFunction),
       () => this.props.location.query.backup,
       () => this.props.jobs,
-      (filteredItems, backup, jobs) => {
+      () => this.props.schedulesByJob,
+      (filteredItems, backup, jobs, schedulesByJob) => {
         if (backup === undefined) {
           return filteredItems
         }
-
         const backedUpVms = uniq(
-          flatMap(jobs, job => filter(filteredItems, createPredicate(omit(job.vms, 'power_state'))))
+          flatMap(
+            filter(jobs, job => some(schedulesByJob[job.id], schedule => schedule.enabled)),
```

Comment:
You can create a selector for "enabled jobs" so this doesn't have to be recomputed every time a VM changes.

Reply:
Maybe even one which returns the predicates?